test/test_json.rb

```diff
@@ -1,6 +1,6 @@
 require_relative "helper"
 require "json"
-require "puma/json"
+require "puma/json_serialization"
 
 class TestJSON < Minitest::Test
```

Comment:
```suggestion
class TestJSONSerialization < Minitest::Test
```

Maybe rename the file too?

Reply:
Ah yes, thanks!

Reply:
New version up -> cf697af